apps/mobile-wallet/src/screens/LandingScreen.tsx

```diff
@@ -108,39 +92,19 @@ const LandingScreen = ({ navigation, ...props }: LandingScreenProps) => {
     navigation.navigate('NewWalletIntroScreen')
   }
 
-  const handleScreenLayoutChange = (e: LayoutChangeEvent) => {
-    const { width, height } = e.nativeEvent.layout
-
-    setDimensions({ width, height })
-  }
-
   return (
     <ThemeProvider theme={themes.dark}>
-      <Screen contrastedBg {...props} onLayout={handleScreenLayoutChange}>
+      <ScrollScreen verticalGap fill headerOptions={{ type: 'stack' }} {...props}>
```

Comment:
Why scroll screen for the landing screen? We don't support scroll there.

Reply:
Does this mean that this will not be addressed in this PR?

Reply:
Yes